orca_detector/vggish/vggish_smoke_test.py

```diff
@@ -41,9 +41,10 @@
 
 print('\nTesting your install of VGGish\n')
 
-# Paths to downloaded VGGish files.
-checkpoint_path = '../data/vggish_model.ckpt'
-pca_params_path = '../data/vggish_pca_params.npz'
+# Paths to downloaded VGGish files. When the Docker container is run
```

Comment:
do we need this file?

Reply:
I don't think so.  Per the other comment thread, I filed an issue to remove it later.